parquet/README.md

```diff
@@ -0,0 +1,33 @@
+# Parquet Sample data
+
+This folder contains sample files in various format
```

Comment:
There needs to be more clarity on what is here and what the purpose is.

Reply:
Added more description with commit 21be2ca